sematic/db/models/resolution.py

```diff
@@ -144,7 +143,7 @@ class ResolutionKind(Enum):
     KUBERNETES = "KUBERNETES"  # for detached mode
 
 
-class Resolution(HasUserMixin, Base, JSONEncodableMixin, HasExternalJobsMixin):
+class Resolution(HasUserMixin, Base, JSONEncodableMixin):
```

Comment:
Doesn't the `external_jobs_json` field need to be removed as well, and the removal of the column propagated to the DB for both `Resolution` and `Run` (if that were possible...)?

Currently both `Resolution` and `Run` reference `Job`, and `Job` references `Run`. This makes the DB denormalized.

Reply:
done